example/package.json

```diff
@@ -10,7 +10,8 @@
   "dependencies": {
     "babel-plugin-module-resolver": "4.1.0",
     "react": "18.1.0",
-    "react-native": "0.70.3"
+    "react-native": "0.70.3",
+    "@animo-id/react-native-ble-didcomm": "^0.1.1"
```

Comment:
The example relies on a symlink of the package `(../)` so we do not have to add it as a dep.

Reply:
I thought so too. But it's currently not written that way (Check `example/App.tsx`). But I will change it now.